src/server/wesnothd/server.cpp

```diff
@@ -1004,7 +1025,7 @@ void server::add_player(socket_ptr socket, const wesnothd::player& player)
 		send_server_message(socket, motd_+'\n'+announcements_, "motd");
```

Comment:
Tournament announcements are sent with the motd now and I'd keep it that way.

Reply:
Updated.